ask-code/ask-code.py

```diff
@@ -24,9 +24,30 @@ def query(question, lsp_brige_port):
     )
 
     # Print the answer
+    prices = {
+        "gpt-4": (0.03, 0.06),
+        "gpt-4-32k": (0.06, 0.12),
+        "gpt-3.5-turbo": (0.0015, 0.002),
+        "gpt-3.5-turbo-16k": (0.003, 0.004),
+        "claude-2": (0.01102, 0.03268),
+        "starchat-alpha": (0.0004, 0.0004),
+        "CodeLlama-34b-Instruct": (0.0008, 0.0008),
+        "llama-2-70b-chat": (0.001, 0.001),
+        "gpt-3.5-turbo-1106": (0.001, 0.002),
+        "gpt-4-1106-preview": (0.01, 0.03),
+        "gpt-4-1106-vision-preview": (0.01, 0.03),
+        "others": (0.001, 0.002),
+    }
```

Comment:
would be better to add a comment noting the first element is the prompt pricing and the second is for completion.